content/post/adding_context_with_annotations.md

```diff
@@ -0,0 +1,106 @@
+---
+author: Michelle Ballen
+title: "Adding Context to Your Analysis with Annotations"
+cover: ""
+publishDate: "2018-07-31"
+draft: true
+tags:
+ - documentation
+ - annotations
+---
+
+Anyone who has worked in digital analytics will tell you that day over day performance can be volatile. Shifts in marketing mix can cause fluctuating e-commerce conversion rates, new feature launches can lead to sudden and temporary swings in engagement rates and onsite bugs can result in anomalies in abandonment rates. Some of these scenarios can be diagnosed through extensive segmentation of data. Others, [like a dropped analytics snippet] (https://www.locallyoptimistic.com/post/web-analytics-process/) or a bug with your payment processor, cannot be so easily uncovered. The simplest thing to do when events like these take place is to take a mental note and count on your memory for when you inevitably have to revisit that data in the future. As you can imagine, taking a mental note isn’t a scalable solution for a number of reasons.
+
+It may seem like an ‘unsexy’ practice relative to other data processes, but from my experience, keeping a record of the online and offline events that affect your business is a practice and well worth the (small) time investment.
+
+<!--more-->
+
+#### Why taking a mental note isn’t a scalable solution:
+* A standard practice among analytics team is to rely on historical data to measure the success of a recent campaign or product launch. If you fail to account for a major event in your ‘pre’ period, it could mean an analysis and recommendation that are not sound.
+* At one point or another, a company executive will most certainly probe you to diagnose an anomaly or shift in performance (read: a dip in performance) -- if you can’t recall all of the events that occurred around the start of the shift there is a good chance that you’ll find yourself going down a rabbit hole try to figure it out.
+* As your company grows, there will undoubtedly be new members joining the analytics team. New analysts can’t possibly be expected to remember major events that they weren’t there to experience. Similarly, veteran employees who were there to experience major events leave the organization at one point or another.
+
+In this post I will walk through the steps I take to integrate annotations into my data models and visualizations. I will refer to an imaginary home goods e-commerce company called “Utensell” to guide the walkthrough.
+
+### Step 1: Whenever something notable happens, record it in a google spreadsheet. Easy!
+
+[screenshot_1]: https://github.com/locallyoptimistic/LocallyOptimistic/blob/master/static/img/annotations-google-sheet.jpg
+
+Examples of things I keep note of:
```

Comment:
> Things to Track:

Reply:
Also you need an additional carriage return to get your bullets to format correctly.